docs/interpreting-experiments/confidence-intervals.md

```diff
@@ -1,34 +1,266 @@
-The conventional way of calculating confidence intervals are based on the Z-test (assuming that there is sufficient data for CLT). That is, a confidence interval for unknown mean $\mu$ based on estimator based on estimator $\hat\mu_n = \bar X$ is found by $(\hat\mu_n - z_{\alpha/2} \hat{\text{SE}},  \hat\mu_n + z_{\alpha/2} \hat{\text{SE}})$, where for $\alpha = 0.05$, $z_{\alpha / 2} = 1.96$. Under the usual assumptions of classical statistical theory, we then have a guarantee that the random interval computed above contains $\mu$ with probability $1-\alpha$.
+[^1]: This is the [frequentist](https://en.wikipedia.org/wiki/Frequentist_inference) interpretation of the significance level. See [below](#bayesian-analysis) for a discussion of what the significance level means if you are using our [Bayesian](https://en.wikipedia.org/wiki/Bayesian_inference) method.
+[^2]: That is, a higher *false positive* or *Type I error* rate.
 
-However, this approach assumes that the sample size is fixed up front, and if this assumption is violated the guarantees for the confidence intervals become invalid.
+## How we calculate the lift estimates and confidence intervals
 
-#### Sequential Analysis
-Eppo uses [sequential analysis](https://arxiv.org/abs/1810.08240) which allows us to compute confidence intervals that are valid across time. More precisely, the probability that the random confidence interval does not contain the unknown mean at any point in time is bounded by $\alpha$. We use the bound in equation (14) of the [reference](https://arxiv.org/abs/1810.08240), and generate confidence intervals by computing
+Eppo provides three methods for estimating the lift and calculating confidence intervals:
 
-$\hat \mu \pm \frac{\sigma}{t} \sqrt{ (t + \rho) \log \left( \frac{t+\rho}{\rho\alpha^2} \right)}$
+1. Fixed-sample frequentist
+2. Sequential frequentist (*the default*)
+3. Bayesian
 
-where $\hat \mu$ is the estimate for the relevant parameter we are interesting in estimating, $t$ is the number of observations, $\alpha$ is the significance level, and $\rho$ is set using
+Each method has advantages and tradeoffs, which we'll discuss below, but we chose to make sequential analysis the default because it allows you the most flexibility about how long to run experiments and when to make decisions, while relying on fewer assumptions that might not hold in common experimentation scenarios. Fixed-sample analysis may in some cases provide a better ability to detect effects, but it does so at the cost of significant restrictions on how you can run your experiments and is particularly brittle to any of the pre-experiment assumptions being incorrect. Bayesian analysis may allow for a more intuitive way to make decisions, since it does not rely on "statistical significance", but requires making pre-experiment assumptions about what the lift is likely to be, which is hard to do in a consistent way for all experiments and for all companies.
```

Comment:
> is particularly brittle to any of the pre-experiment assumptions being incorrect

This seems vague; I imagine you are trying to say that you have to specify run time in advance and you need to set the MDE correctly. Maybe we can state that more explicitly?

Reply:
Or we could reword entirely and say: if you have a culture of running experiments for a fixed amount of time, e.g. 2 weeks, and you avoid peeking and deciding prematurely, then the fixed sample test gives you more power.